### PR TITLE
change order from dropdown to arrow up and down icon on table head

### PR DESCRIPTION
**Pull request by Muh-Baharuddin**
@sidratul please reviwe my code

### CODE REVIEW

File: src/containers/Supplier/Table/TableSupplier.tsx [hunk 1]
```diff
@@ -87,30 +87,36 @@ const TableSupplier = () => {
...
-                >
-                  <i className="bi bi-arrow-down-up"></i>
-                </span>
+                {queryParams.orderBy === 'nama' && (
+                  <i
+                    className={`bi bi-arrow-${
+                      queryParams.orderType === 'ASC' ? 'down' : 'up'
+                    }-square`}
+                  ></i>
+                )}
```
Comment (Muh-Baharuddin) [inline src/containers/Supplier/Table/TableSupplier.tsx:99]: Mar-Fashion-Web-order-down

File: src/containers/Supplier/Table/TableSupplier.tsx [hunk 2]
```diff
@@ -87,30 +87,36 @@ const TableSupplier = () => {
...
-              <th>
+              <th onClick={() => handleSortBy('alamat')}>
                 Alamat{' '}
-                <span onClick={() => handleSortBy('alamat')}
-                  style={{ cursor: 'pointer' }}
-                >
-                  <i className="bi bi-arrow-down-up"></i>
-                </span>
+                {queryParams.orderBy === 'alamat' && (
+                  <i
```
Comment (Muh-Baharuddin) [inline src/containers/Supplier/Table/TableSupplier.tsx:104]: Mar-Fashion-Web-order-alamat

File: src/containers/Supplier/Table/TableSupplier.tsx [hunk 3]
```diff
@@ -92,29 +92,29 @@ const TableSupplier = () => {
...
+                    }-fill`}
                   ></i>
                 )}
               </th>
               <th onClick={() => handleSortBy('nomor_telepon')}>
                 Nomor Telepon{' '}
                 {queryParams.orderBy === 'nomor_telepon' && (
                   <i
-                    className={`bi bi-arrow-${
+                    className={`bi bi-caret-${
```
Comment (Muh-Baharuddin) [inline src/containers/Supplier/Table/TableSupplier.tsx:115]: Mar-Fashion-Web-caret

**Review by sidratul — APPROVED** [review]

**Pull request merged**